framework/include/fvkernels/FVFluxKernel.h

```diff
@@ -124,10 +124,17 @@ class FVFluxKernel : public FVKernel,
       Moose::FV::LimiterType limiter_type = Moose::FV::LimiterType::CentralDifference,
       bool correct_skewness = false) const;
 
-  const bool _force_boundary_execution;
-
-  std::unordered_set<BoundaryID> _boundaries_to_force;
-  std::unordered_set<BoundaryID> _boundaries_to_not_force;
+  /**
+   * Returns whether to avoid execution on a boundary
+   * @param fi the FaceInformation currently considered
+   */
+  bool avoidBoundary(const FaceInfo & fi) const
+  {
+    for (const auto bnd_id : fi.boundaryIDs())
+      if (_boundaries_to_avoid.count(bnd_id))
+        return true;
+    return false;
```

Comment:
put this in .C or make it `inline` outside of the class definition

Reply:
moved in .C